### DISCUSSION TITLE
insufficient memory

### DISCUSSION THREAD

**Discussion by thmme** [sepinf-inc/IPED#831] [Q&A] [answered]
Dear all, 

I have a issue to processing a .E01 file using the version iped-3.18.9. 

the processing not started and show a message error "insufficient memory available: below 200MB.... increase heap memory with -Xmx java option or decrease 'numthreads' in advanced configuration. 

I tried several ways to set the file AdvancedConfig.txt, but did not work well. 

PC settings: 8 Gbytes RAM.

Regards,

TM

**Comment by wladimirleite** [comment, marked as answer]
How many cores are being used?
Check if you are using a 64-bit JRE.
I think the number of processing threads used can be changed in LocalConfig.txt.

**Comment by wladimirleite** [comment]
There are other tips that should help in Discussions:
https://github.com/sepinf-inc/IPED/wiki/Performance-Tips

**Reply by thmme** [reply]
Hi, thank you!! it worked well!!

**Reply by lfcnassif** [reply]
Could you specify what worked? Maybe it could help other users.

**Reply by EstephanoBartenski** [reply]
I'm having the exact same issue. I'm using IPEDIAR so I don't know how to use codes in order to specify an amount of JVM. Can I solve this by changing the numthreads? To what value? Currently trying IPED without a SSD (just learning for now), using a 8 GB RAM. This problem appears only when trying to generate a report after an analysis. Tks in avance

**Reply by lfcnassif** [reply]
I recommend to use a computer with more memory. Anyway, to try to workaround your problem to generate a report, go to the case folder, iped/LocalConfig.txt and set numThreads to half the number of CPU cores you have, then try to generate the report again.

**Reply by EstephanoBartenski** [reply]
Hi there! What's a good specification of a computer to use? Not the minimun, but one that wouldnt have any running issues

**Reply by lfcnassif** [reply]
There are some useful tips here: https://github.com/sepinf-inc/IPED/wiki/Performance-Tips

**Comment by lfcnassif** [comment]
> in advanced configuration.

This message should be updated as Wladimir pointed out.

Maybe using
java -Xms4G -Xmx4G -jar iped.jar...
together could help.

**Comment by lfcnassif** [comment]
I'm moving this to discussions
